lib/vanagon/engine/local.rb

```diff
@@ -1,40 +1,38 @@
+require 'vanagon/engine/base'
 require 'vanagon/utilities'
 require 'vanagon/errors'
-require 'benchmark'
 
 class Vanagon
   class Engine
-    class Local
-      attr_accessor :target
+    class Local < Base
 
       def initialize(platform, target = nil)
         @platform = platform
         @target = "local machine"
         @name = 'local'
+        super
+        @required_attributes = Array.new
       end
 
       # Dispatches the command for execution
-      def dispatch(command)
-        puts Benchmark.measure { local_command(command, @workdir) }
-      end
-
-      # Steps needed to tear down or clean up the system after the build is
-      # complete
-      def teardown
-      end
-
-      # This method will take care of validation and target selection all at
-      # once as an easy shorthand to call from the driver
-      def startup(workdir)
-        @workdir = workdir
-        script = @platform.provisioning.join(' && ')
-        dispatch(script)
+      def dispatch(command, return_output = false)
+        # unset environment variables set by 'bundle exec build'
```

Comment:
Using this (`defined?(Bundler)`) seems to work. @pyther can you try it this way and validate that it matches your expectations? I'll review the rest of your PR tomorrow (I think we need to update some yardoc to match new inputs for updated functions); I can help you with that.

Thanks for taking the time to exercise this particular code path! Your help is appreciated.